test/runtests.jl

```diff
@@ -21,6 +21,8 @@ eltypes = (Int64, Float64, BigFloat)
             @test S == S
             @test copy(S) == S
             @test !(copy(S) === S)
+            @test typeof(similar(S)) == typeof(S)
+            (t != BigFloat) && @test (S2 = similar(S); copy!(S2, S); S2 == S)
```

Comment:
The check here is necessary for the 1.0 tests to pass, since apparently there is no `copy!` for BigFloats. We could keep it like this, or just decide to not test for 1.0 and instead rather test for 1.6, as proposed in #15.

Reply:
Ok so the tests still fail, apparently there is no `copy!` for Int64?? Overall, I'm seriously considering dropping 1.0 for 1.6. What do you think @pnkraemer?

Reply:
I am fine with that :) We can revive #15?! (Or start a new PR, which might be easier)